Description: Don't exit in case of a failed memory allocation
 This patch modifies src/xmalloc.c to not exit in case of a
 failure of memory allocation. This fixes lintian X: tag
 shlib-calls-exit.
Author: Bilal Akhtar <bilalakhtar@ubuntu.com>
--- a/src/xmalloc.c
+++ b/src/xmalloc.c
@@ -30,7 +30,7 @@
 static void *xmalloc_fatal(size_t size) {
   if (size==0) return NULL;
   fprintf(stderr, "Out of memory.");
-  exit(1);
+  return NULL;
 }
 
 void *xmalloc (size_t size) {
